int8 move
uint8 speed1
uint8 turn
